language-support/ts/daml-ledger/index.ts

```diff
@@ -586,7 +607,9 @@ class Ledger {
     template: Template<T, K, I>,
     key: K,
   ): Stream<T, K, I, CreateEvent<T, K, I> | null> {
+    let lastContractId: ContractId<T> | null = null;
     const request = [{templateId: template.templateId, key}];
+    const reconnectRequest = (offset: string): object[] => [{'offset': offset}, {...request[0], 'contractIdAtOffset': lastContractId}]
```

Comment:
Does this actually work? I think it has to come as two consecutive messages.

Reply:
I interpreted the documentation around this like this. But now that you asked, I'm not sure. Is there a  way to trigger a connection close in the JSON API server?

Reply:
I checked back with the runtime team. Apparently, it needs to be two messages. I changed it accordingly.